bin/bundle-accp-jdk8.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -e
+set -x
+
+# Use this script for JDKs 8. Please note that this script does not modify the security file,
+# so one still needs to programmatically install ACCP.
+#
+# This script requires JAVA_HOME set properly.
+#
+# usage: ./bundle-script 2.3.3 linux-x86_64
+
+ACCP_VERSION=$1 # for example 2.3.3
+ACCP_PLATFORM=$2 # for example linux-x86_64 or osx-aarch64
```

Comment:
nit: should we fail out with an error if nothing here is specified or default to something reasonable (perhaps grepping for latest version and using `uname` for `$1` and `$2` respectively)?

Reply:
Depending on the audience, I agree that more input sanitizing may be helpful to the user, including:

* Check that the third argument is `-FIPS` or not, and error otherwise. This is because people can easily switch order of argument and this would result in hard to understand errors.
* Check number of arguments is 2 or 3 and display usage if not valid.
* Maybe check that ACCP_PLATFORM is in a list? Another option is to have an error message when downloading fails asking to check the version/platform/FIPS.
* Check that JAVA_HOME is not empty and is a folder.

Reply:
If any of these options is not set properly, the URL for download wouldn't be valid and download would fail. These scripts are provided as examples.